src/coreclr/vm/frames.h

```diff
@@ -445,16 +428,11 @@ class Frame
         TYPE_INTERNAL,
         TYPE_ENTRY,
         TYPE_EXIT,
-        TYPE_CONTEXT_CROSS,
         TYPE_INTERCEPTION,
         TYPE_SECURITY,
```

Comment:
`TYPE_SECURITY` is unused as well

Reply:
Those are used:

```
src/coreclr/debug/ee/controller.cpp:            else if (info->m_activeFrame.frame->GetFrameType() == Frame::TYPE_SECURITY &&
src/coreclr/debug/ee/frameinfo.cpp:        case Frame::TYPE_SECURITY: // Security is a sub-type of interception
src/coreclr/debug/ee/frameinfo.cpp:            LOG((LF_CORDB, LL_INFO100000, "DWSP: Frame type is TYPE_INTERCEPTION/TYPE_SECURITY.\n"));
src/coreclr/vm/frames.h:        TYPE_SECURITY,
```

Reply:
`TYPE_SECURITY` is never returned by `GetFrameType`, so `GetFrameType() == Frame::TYPE_SECURITY` is always false. Similar for the rest.